CIScripts/Tests/AgentServiceTests.ps1

```diff
@@ -0,0 +1,255 @@
+function Test-AgentService {
+    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session,
+           [Parameter(Mandatory = $true)] [TestConfiguration] $TestConfiguration)
+
+    . $PSScriptRoot\CommonTestCode.ps1
+    . $PSScriptRoot\..\Job.ps1
+
+    $AgentServiceTestsTimeTracker = [Job]::new("Test-AgentService")
```

Comment:
I think that we should not create new TimeTracker here. We should use existing Job like in this commit https://github.com/codilime/contrail-windows-tools/commit/962af2d37b14dc4a94448d381e279ffc496119f3